src/sql/workbench/browser/editor/profiler/dashboardInput.ts

```diff
@@ -93,6 +94,28 @@ export class DashboardInput extends EditorInput {
 		return name;
 	}
 
+	public override getTitle(verbosity?: Verbosity): string {
```

Comment:
Where is this method used? Can this be made abstract instead, if the implementation here isn't been used anywhere else?

Reply:
This method is used by tabsTitleControl under src/vs/workbench/browser/parts/editor to either display the top title as short form, and the long title as the tooltip. It is part of the vscode EditorInput interface and should not be overridden as a lot of editors in vscode implement it this way.